helm/README.md

```diff
@@ -41,6 +41,12 @@ The following table lists the configurable parameters of the Nessie chart and th
 | mongodb | Configuration specific to `versionStoreType: MONGO` | |
 | mongodb.name | The database name | `nessie` |
 | mongodb.connectionString | The database connection string | `mongodb://localhost:27017` |
+| postgres | Configuration specific to `versionStoreType: TRANSACTIONAL` | |
+| postgres.jdbcUrl | The database connection URL | `jdbc:postgresql://localhost:5432/my_database` |
+| postgres.secret.name | The name of the secret where database credentials are stored | `postgres-creds` |
+| postgres.secret.username | The name of the username key inside the secret | `postgres_username` |
+| postgres.secret.password | The name of the password key inside the secret | `postgres_password` |
```

Comment:
see https://github.com/projectnessie/nessie/pull/4223#discussion_r873385806, let's please also add a section below with:
### Providing secrets for Postgres

* Make sure you have a Secret in the following form:
```
> cat $PWD/postgres-creds
postgres_username=YOUR_USERNAME
postgres_password=YOUR_PASSWORD
```